games/h.yaml

```
@@ -59,6 +59,22 @@
   status: unplayable
   type: clone
   updated: 2016-12-11
+  
+- name: Greentwip's Harvest Moon
+  info: Harvest Moon (SNES) remake for UWP and Xbox One.
+  lang:
+  - C#
+  license:
+  - GPL3
+  development: active
+  originals:
+  - Story of Seasons series
+  repo: https://github.com/greentwip/HarvestMoon
+  status: playable
```

```suggestion
  status: semi-playable
```
Looks like gameplay features are missing compared to the original, so this is more appropriate